CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog
+## v5.3.4 (Not yet published)
+
+### Added
+
+-   Added class `content` to `<EmptyState>`.
+-   Added padding to `<EmptyState>`.
```

Comment:
Please specify the padding; added `1rem` padding

Reply:
Is there any reason for not being able to add the padding on the root class instead?

Reply:
Root div element is taking height of window, so I added one more div which takes height of content and added flex.